examples/blender/blender.py

```diff
@@ -75,9 +75,17 @@ async def worker(ctx: WorkContext, tasks):
             script.download_file(f"/golem/output/out{frame:04d}.png", output_file)
             try:
                 yield script
-                # TODO: Check if job results are valid
-                # and reject by: task.reject_task(reason = 'invalid file')
-                task.accept_result(result=output_file)
+
```

Comment:
I know this is not the master branch but maybe it would be good for this part to depend on some parameter, e.g. `--randomize-task-acceptance` ? ...